mllib/src/main/scala/org/apache/spark/mllib/optimization/GradientDescent.scala

```diff
@@ -17,11 +17,13 @@
 
 package org.apache.spark.mllib.optimization
 
+import collection.JavaConverters._
```

Comment:
Nit: scala.collection.JavaConverters._ but where is this used? and in the next file.